adminer/include/editing.inc.php

```diff
@@ -151,14 +151,14 @@ function edit_type($key, $field, $collations, $foreign_keys = array(), $extra_ty
 	$structured_types[lang('Foreign keys')] = $foreign_keys;
 }
 echo optionlist(array_merge($extra_types, $structured_types), $type);
-?></select>
-<?php echo on_help("getTarget(event).value", 1); ?>
-<?php echo script("mixin(qsl('select'), {onfocus: function () { lastType = selectValue(this); }, onchange: editingTypeChange});", ""); ?>
+?></select><!--
```

Comment:
You have not correctly written styles.
```
<table class=nowrap>
.nowrap {
    white-space: pre;
}
```
I does't change code, I added <!-- --> because it is falling apart.

Reply:
I don't see anything falling apart. If there's a bug showing in some browser then resolve it in a separate pull request.

Reply:
fix in new commits.